man/fi_peer.3.md

```diff
@@ -286,6 +289,51 @@ on the local node.
    creates a reference between the peer_cq object and its own cq.
 ```
 
+# PEER COUNTER
+
+The peer counter defines a mechanism by which a peer provider may increment
+value or error into the counter owned by another provider.
+
+The setup of a peer counter is similar to the setup for a peer CQ outlined
+above. The owner’s counter object is imported directly into the peer.
+
+The data structures to support peer counters are defined as follows:
+
+```c
+struct fi_ops_cntr_owner {
+    size_t  size;
+    ssize_t (*inc)(struct fid_peer_cntr *cntr);
+    ssize_t (*incerr)(struct fid_peer_cntr *cntr);
```

Comment:
The API addition in the previous patch has these returning void but here it says ssize_t. I think void is fine.

Reply:
Good catch, they should be void. I will fix it.